github/orgs_teams.go

```diff
@@ -41,6 +44,19 @@ func (t Team) String() string {
 	return Stringify(t)
 }
 
+// Invitation represents a team member's inviation status
+type Invitation struct {
+	ID        *int       `json:"id,omitempty"`
+	Login     *string    `json:"login,omitempty"`
+	Email     *string    `json:"email,omitempty"`
+	Role      *string    `json:"role,omitempty"`
+	CreatedAt *time.Time `json:"created_at,omitempty"`
```

Comment:
What's the source for the `created_at` field? I don't see it in the example at https://developer.github.com/v3/activity/events/types/#organizationevent.